projects/concurrency/concurrency_test.go

```diff
@@ -72,20 +71,33 @@ func TestConcurrency(t *testing.T) {
 
 	t.Run("concurrent reads and writes", func(t *testing.T) {
 		c := NewCache[string, any](3)
-		
+		c.Put("occupation", "Pilot")
+
 		var wg sync.WaitGroup
 
-		for i:=0; i < 100; i++ {
+		for i := 0; i < 100; i++ {
 			wg.Add(1)
-			go func () {
+			go func() {
 				value := fmt.Sprintf("Pilot%d", i)
 				c.Put("occupation", value)
 				wg.Done()
 			}()
-			
+
 			wg.Add(1)
-			go func () {
-				c.Get("occupation")
+			go func() {
+				value, ok := c.Get("occupation")
+				if !ok {
+					t.Errorf("expected %v but got %v", true, ok)
+				}
+				if value == nil {
+					t.Errorf("expect value to not be nil")
+				} else {
+					stringifiedValue := (*value).(string)
+					got := strings.Split(stringifiedValue, "")[0:5]
+					if strings.Join(got, "") != "Pilot" {
```

Comment:
This is quite a fiddly way to check the first 5 characters of a string... Can you think of a simpler way?